internal/cephfs/store/volumeoptions.go

```diff
@@ -209,10 +209,32 @@ func fmtBackingSnapshotOptionMismatch(optName, expected, actual string) error {
 		optName, actual, expected)
 }
 
+// getVolumeOptions validates the basic required basic options provided in the
```

Comment:
I wonder where this function comes from? Was it maybe included through rebasing on an older version of the `devel` branch?